static/css/style.css

```diff
@@ -1190,3 +1190,11 @@ code {
   text-indent: 20px;
   display: block;
 }
+
+.rotate90 {
```

Comment:
I would much rather rotate the actual image rather than do this css hack. Let me know if you need assistance with that.

Reply:
Yes, can you please help me with that, this looks like a hack. I tried to tweak with the image itself but due to its pixels, it seems to not be oriented correctly. Or do you suggest we can remove that picture too!

Reply:
Fixed, pushed to your branch and also resized all of the images so they are lighter.